### PR TITLE
Completed javascript-loops exercise.

### PR DESCRIPTION
**Pull request by mkrtchyan-tigran**

### CODE REVIEW

File: javascript-loops/main.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+function getNumbersToTen() {
+  var numbers = [];
+  var currentNumber = 1;
+  while (currentNumber <= 10) {
+    numbers.push(currentNumber);
+    currentNumber++;
+  }
+  return numbers;
+}
+getNumbersToTen();
```
Comment (scrumptiousdev) [inline javascript-loops/main.js]: You don't need to call/invoke functions in this file — please remove all the function calls

File: javascript-loops/main.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    repeated += word;
+    count++;
+  }
+  return repeated;
+}
+repeatWord();
+
+function logEachCharacter(string) {
+  for (var i = 0; i < string.length; i++) {
+    console.log(string);
```
Comment (scrumptiousdev) [inline javascript-loops/main.js]: You need to log "each" character like: `j`, `a`, `v`, `a` and so on.

**Review by scrumptiousdev — CHANGES_REQUESTED** [review]

**Review by scrumptiousdev — APPROVED** [review]
Approved!

File: javascript-loops/main.js (hunk 2 above)
Comment (mkrtchyan-tigran) [reply javascript-loops/main.js]: Forgot to put `[i]` :(

**Pull request merged**